controllers/profiles/object_creators.go

```diff
@@ -42,10 +42,26 @@ const (
 
 	workflowConfigMapNameSuffix      = "-props"
 	configMapWorkflowPropsVolumeName = "workflow-properties"
+
+	// Quarkus Health Check Probe configuration.
+	// See: https://quarkus.io/guides/smallrye-health#running-the-health-check
+
+	quarkusHealthPathStarted = "/q/health/started"
+	quarkusHealthPathReady   = "/q/health/ready"
+	quarkusHealthPathLive    = "/q/health/live"
+
+	// Default deployment health check configuration
+	// See: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+
+	healthTimeoutSeconds             = 3
+	healthStartedFailureThreshold    = 5
+	healthStartedPeriodSeconds       = 15
+	healthStartedInitialDelaySeconds = 10
 )
 
 var defaultApplicationProperties = "quarkus.http.port=" + strconv.Itoa(defaultHTTPWorkflowPort) + "\n" +
-	"quarkus.http.host=0.0.0.0\n"
+	"quarkus.http.host=0.0.0.0\n" +
+	"org.kie.kogito.addons.knative.health-enabled=false\n"
```

Comment:
why disabling knative health by default ? maybe a comment next to it ?

Reply:
Because in dev, we won't block the pod to start if Knative bindings are not available. I'll add a comment and change this property to `defaultDevApplicationProperties` to make it clear.